mesos-marathon/centos/README.md

```diff
@@ -1,34 +1,138 @@
 ---
 layout: guides
-title: "Using Weave with Mesos Marathon on CentOS"
+title: "Using Weave with Apache Mesos & Marathon on CentOS"
 permalink: /guides/platform/mesos-marathon/os/centos/cloud/vagrant/index.html
 tags: docker, mesos, marathon, cli, vagrant, virtualbox, dns, ipam
 ---
 
-In this guide I will demonstrate how to get Weave working with Mesos Marathon on CentOS 7 using Vagrant.
+##What You Will Build## 
+
+Weave provides a software network that is optimized for visualizing and communicating with apps scattered within Docker containers. Using tools and protocols that are familiar to you, Weave provides the network topology that allows you to communicate between containerized apps distributed across multiple networks or hosts more quickly and efficiently.
+ 
+This example describes how to setup a Weave Network on a cluster manager, specifically, within the Apache Mesos & Marathon environment. For more details on the Apache Mesosphere and its application manager, Marathon, see the [Apache Mesos Docs](http://mesos.apache.org/documentation/latest/) and the [Marathon Docs](https://mesosphere.github.io/marathon/).
+
+##What You Will Use ##
+
+* [Weave](http://weave.works)
+* [Docker](http://docker.com)
+* [CentOS](http://http://centos.org/)
+* [Apache Mesos Marathon](https://mesosphere.github.io/marathon/)
+
+## Before you start ##
+
+This self-contained tutorial uses Vagrant to install and configure the virtual machines with CentOS, then it sets up and configures Apache Mesos Marathon and finally it installs and launches both Docker and Weave onto the virtual machines.  
+
+Before you start have the following installed: 
+
+* [Git](http://git-scm.com/downloads)
+* [Vagrant > 1.6](https://docs.vagrantup.com/v2/installation/index.html)
+* [VirtualBox > 4.3.20](https://www.virtualbox.org/wiki/Downloads) 
+
 
 ## Let's go!
 
-First, you need to clone the repository
+1.  Clone the repository:
+
+        git clone https://github.com/weaveworks/guides
+     						
+2.  Change to the following sub-directory: 
+
+        cd mesos-marathon/centos
+    
+3.	Run the Vagrant script: 
+
+        vagrant up
+        
+4. The Vagrant script installed the Virtual Machines `mesos-00` and `mesos-01`. `mesos-00` is configured as a 	master, where it runs as a single Zookeeper instance within the Marathon framework. `mesos-01` is configured in this example, as the slave.
```

Comment:
Zookeeper is a key dependency of Mesos, all of the components (master and slave) talk to it. Marathon is a framework on top of it all.